app/security_system.py

```diff
@@ -82,7 +90,10 @@ def log_access_attempt(self, person_name, is_authorized):
         # Logs should be in the following format:
         # <timestamp> - Person: <person_name>, Authorized: <is_authorized>
         # Your code goes here
-
+        
+        timestamp = time.time()
+        logging.info(f'{timestamp} - Person: {person_name}, Authorized: {is_authorized}')
+        
```

Comment:
If your logger is already configured, then we don't need to include a timestamp here. It will automatically log in the specified format.